### DISCUSSION TITLE
Einstellung Open DTU on Battery und Shelly 3M

### DISCUSSION THREAD

**Discussion by BMGRalf** [hoylabs/OpenDTU-OnBattery#987] [General]
Hallo,

ich habe mich jetzt ganz neu mit dem Thema Shelly 3M und Open DTU on Battery befasst. Die OpenDTU habe ich soweit eingerichtet das ich die Daten vom Wechselrichter bekomme. Ich bekomme allerdings keine Kommunikation mit der Shelly 3M zustande.
Da ich mich hiermit gar nicht auskenne, hätte ich gerne die Bitte das mir jemand dabei helfen kann.

Hat jemand eventuell Screenshots von den Einstellungen?

Über Antworten wäre ich sehr dankbar.

Gruß BMGRalf

**Comment by spcqike** [comment]
Nabend 

hast du schon mal im Wiki geguckt? 

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Power-Meter

**Comment by BMGRalf** [comment]
Danke.
Das hat gut funktioniert. Jetzt sehe ich die Daten in der OpenDTU. Werde mich jetzt mal einlesen wie ich die Nulleinspeisung mache.
Dafür habe ich schon einige Sachen gesehen.
